dart/lib/src/sentry_options.dart

```diff
@@ -254,5 +255,10 @@ void noOpLogger(SentryLevel level, String message) {}
 
 /// A Logger that prints out the level and message
 void dartLogger(SentryLevel level, String message) {
```

Comment:
For v6 I'd like to add an error and stacktrace parameter to our loggers.

They can be passed to https://api.flutter.dev/flutter/dart-developer/log.html and look hopefully better